docs/getting-started/integration.md

```diff
@@ -34,3 +34,23 @@ require(['path/to/chartjs/dist/Chart.js'], function(Chart){
 ```
 
 > **Important:** RequireJS [can **not** load CommonJS module as is](http://www.requirejs.org/docs/commonjs.html#intro), so be sure to require one of the built UMD files instead (i.e. `dist/Chart.js`, `dist/Chart.min.js`, etc.).
+
+## Content Security Policy (CSP)
```

Comment:
It might make sense to add an extra `#` in the four headers above this. They're all about including the JS and this one is about including the CSS, so it may make sense to differentiate somehow